src/codelabs/dart-cheatsheet_migrated.md

```diff
@@ -439,35 +508,35 @@ bool hasEmpty = aListOfStrings.any((s) => s.isEmpty);
 
 Try finishing the following statements, which use arrow syntax.
 
-```dart:run-dartpad:height-345px:ga_id-arrow_syntax
+```dart:run-dartpad:height-345px:ga_id-arrow_syntax:null_safety-true
 {$ begin main.dart $}
 class MyClass {
-  int _value1 = 2;
```

Comment:
It seems to me that the variables are intentionally private, although underscore and its relationship to privacy are never discussed. (Whoops. I'll try to remedy that.)

Do you think there's a good reason to make these variables public?

Reply:
Actually, on second thought, I think there's no good reason to use `_` here. But I should still add something about privacy and underscore, probably in "Getters and setters".

Reply:
I'll just leave a comment here in favor of the private fields. This class seems to intentionally encapsulate some of the behavior with a method to increment a specific value, calculate a product, join them, etc with the intention to base it off of those fields. 

While I know this is purely a theoretical class created as an example, if a user of this object could manipulate those freely, it seems like it would be unexpected/unintended by the writer. Why else would they have the form of encapsulation.

Reply:
The reason I changed the visibility of the fields is because it would show a warning  in the dartpad otherwise:
![image](https://user-images.githubusercontent.com/2494376/105468368-32fa1780-5c97-11eb-9961-664c1603c282.png)

As most other examples in the codelab also used public fields, I thought it would be a good idea to make them public as well.

I could switch back to private fields and then have a couple of statements in the test section that could use them, so there's no warnings in the dartpad. Or we could simple tell the learner to ignore the warnings.

Let me know what you prefer! :)